crates/interpreter/src/interpreter/shared_context.rs

```diff
@@ -0,0 +1,56 @@
+use crate::{SharedMemory, SharedStack, EMPTY_SHARED_MEMORY, EMPTY_SHARED_STACK};
+
+pub const EMPTY_SHARED_CONTEXT: SharedContext = SharedContext {
+    stack: EMPTY_SHARED_STACK,
+    memory: EMPTY_SHARED_MEMORY,
+};
```

Comment:
This should be a `const fn empty() -> Self { ... }` or `const EMPTY: Self = ...;` on all the related structs. Const item initializers are weird to me, so I'd prefer the former.

Reply:
Okay! I kept this for consistency with rakita's work, if he also agrees with this change for me it's perfectly fine